pyci/utility.py

```diff
@@ -174,10 +176,19 @@ def odometer_one_spin(wfn, cost, t, qmax):
     # Index of last particle
     j = wfn.nocc_up - 1
     # Select determinants
+    if isinstance(wfn, pyci.two_spin_wfn):
```

Comment:
I don't think we need these checks in here.
There are 2 algorithms: `odometer_one_spin` and `odometer_two_spin` that are used for one_spin and two_spin wave functions